src/EditorFeatures/Core/FindUsages/ClassifiedSpansAndHighlightSpanFactory.cs

```diff
@@ -100,12 +95,43 @@ private static TextSpan GetLineSpanForReference(SourceText sourceText, TextSpan
                 new ClassifiedSpan(ClassificationTypeNames.Text, narrowSpan),
                 new ClassifiedSpan(ClassificationTypeNames.Text, TextSpan.FromBounds(narrowSpan.End, widenedSpan.End)));
         }
+    }
+
+
+    internal static class EditorClassifier
```

Comment:
this commit just extracts the code above into EditorClassifier.  The next commit moves it into its own file.